payments-ui-core/res/values/strings.xml

```diff
@@ -80,6 +80,8 @@
   <string name="scan_card">Scan card</string>
   <!-- SEPA mandate text -->
   <string name="sepa_mandate">By providing your payment information and confirming this payment, you authorise (A) %s and Stripe, our payment service provider, to send instructions to your bank to debit your account and (B) your bank to debit your account in accordance with those instructions. As part of your rights, you are entitled to a refund from your bank under the terms and conditions of your agreement with your bank. A refund must be claimed within 8 weeks starting from the date on which your account was debited. Your rights are explained in a statement that you can obtain from your bank. You agree to receive notifications for future debits up to 2 days before they occur.</string>
+  <!-- Cash App Pay mandate text -->
+  <string name="cashapp_mandate">By continuing, you authorize %1$s to debit your Cash App account for this payment and future payments in accordance with %2$s\'s terms, until this authorization is revoked. You can change this anytime in your Cash App Settings.</string>
```

Comment:
Looks like this is only used with 1 string. Should we make both instances `%1$s`. Then you can revert some other changes in the PR. Otherwise I think we should test the other new changes related to this.

Reply:
I removed the changes, because we ended up not supporting PI+SFU and SI for Cash App Pay in Payment Sheet.